client/api_object.go

```diff
@@ -654,8 +687,7 @@ func (c *client) GetObjectUploadProgress(ctx context.Context, bucketName, object
 	}
 
 	// get object status from sp
-	if status.ObjectStatus == storageTypes.OBJECT_STATUS_CREATED &&
-		status.ObjectStatus != storageTypes.OBJECT_STATUS_SEALED {
```

Comment:
Why delete this line?

Reply:
status.ObjectStatus != storageTypes.OBJECT_STATUS_SEALED  is useless